src/components/MoviesList/MoviesList.jsx

```diff
@@ -1,10 +1,26 @@
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
-export const MoviesList = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList = ({ movies, query }) => {
+  const trimmQuery = query.trim().toLowerCase();
+
+  const filteredMovies = [...movies].filter((movie) => {
+    const title = movie.title
+      .toLowerCase()
+      .includes(trimmQuery);
+
+    const description = movie.description
+      .toLowerCase()
+      .includes(trimmQuery);
+
+    return title || description;
+  });
```

Comment:
1. Not need to make array copy, because filter method return new array
2. Please move this function to App.jsx outside the component, because it would be more correct to pass an already filtered array to this component
```suggestion
  const filteredMovies = movies.filter((movie) => {
    const title = movie.title
      .toLowerCase()
      .includes(trimmQuery);

    const description = movie.description
      .toLowerCase()
      .includes(trimmQuery);

    return title || description;
  });
```